doc/src/create_atoms.txt

```diff
@@ -242,9 +242,8 @@ write_dump      all atom sinusoid.lammpstrj :pre
 
 :c,image(JPG/sinusoid_small.jpg,JPG/sinusoid.jpg)
 
-The {rotate} keyword can only be used with the {single} style and
-when adding a single molecule. It allows to specify the orientation
-at which the molecule is inserted.  The axis of rotation is
+The {rotate} keyword allows to specify the orientation
```

Comment:
please change "to specify" to "specification of"

Reply:
once this grammar is fixed (was also in the original), this
is ready to merge

Reply:
I've fixed the error.